doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -468,7 +468,7 @@ checks and is only meant to improve readability of the call.
 
 ### Struct Type correspondences
 
-Composite types, aka `struct` in C or `TYPE` in Fortran90 (or `STRUCTURE` / `RECORD` in some variants
+Composite types, e.g., `struct` in C or `TYPE` in Fortran90 (or `STRUCTURE` / `RECORD` in some variants
```

Comment:
probably "such as" would be clearer here than "e.g.,"

Reply:
I think I've done the minor changes to the minor changes now (except for one where noted I think).